0359-ccache-storage-service.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 📦 ccache remote storage
+aliases: [ ]
+---
+
+While `cmake-re` is optimized for CMake-based builds, it can also operate as a compiler, linker and archiver driver — making it possible to integrate with `ccache` to provide an RE-API backend as remote shared cache.
+
+Once installed `cmake-re` will provide the following tools as part of it's distribution folder.
+
+- `tipi-compiler-driver`
+- `tipi-linker-driver`
+- `tipi-ar-driver`
+- `tipi-ranlib-driver`
+
+## RE-API instead of ccache `remote_storage`
+`ccache` supports it's own remote storage backend, our remote caching for ccache doesn't use this abstraction and instead relies on the more complete [Bazel RE-API](https://github.com/bazelbuild/remote-apis).
```

Comment:
Typo: its own

We don't have "remote caching for ccache", we have a caching solution that can be layered over ccache, and it is using the RE-API.
You can then add that it will provide both a fast local cache and a fast remote cache solution that can be shared by a whole team etc.

Reply:
I would not user layers as it's overloaded in our documentation with L1 and L2 layers and docker layers and toolchains layer files etc.

But I made it clearer that it's a combination.